docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -2236,6 +2236,20 @@ How to **fix** it:
 
 - Increase the allowed limit by using the `-distributor.max-recv-msg-size` option.
 
+### err-mimir-distributor-max-otlp-request-size
+
+This error occurs when a distributor rejects an Otel write request because its message size is larger than the allowed limit after decompression.
```

Comment:
The size is checked also prior to decompressing, right:
```suggestion
This error occurs when a distributor rejects an OTel write request because its message size is larger than the allowed limit before or after decompression.
```